.travis.yml

```diff
@@ -9,3 +9,4 @@ before_install:
 script:
   - gometalinter ./... --fast -D dupl -D gas -D goconst
   - go test -v ./...
+  - cd ./cmd/seabird && go build
```

Comment:
This will force a seabird build with all plugins, so we see build errors like the uuid one earlier.